tests/test-sources/plugins/git/lazygit.nix

```diff
@@ -0,0 +1,21 @@
+{
+  empty = {
+    plugins.lazygit.enable = true;
+  };
+
+  defaults = {
+    plugins.lazygit = {
+      enable = true;
+
+      settings = {
+        floating_window_winblend = 0;
+        floating_window_scaling_factor = 0.9;
+        floating_window_border_chars = ["╭" "─" "╮" "│" "╯" "─" "╰" "│"];
+        floating_window_use_plenary = false;
+        use_neovim_remote = true;
+        use_custom_config_file_path = false;
+        config_file_path = "";
```

Comment:
```suggestion
        config_file_path = [];
```

Reply:
done (and I also modified the settingsExample for coherency)